README.md

```diff
@@ -12,104 +12,44 @@ npm install --save simplified-json
 
 ## Usage
 
-`sample.sjson`
```

Comment:
Yes I could add some small sample.  I just felt like the actual code used to get lost in the big samples.

Would it also be worth it to add a link to [ SJSON](http://help.autodesk.com/view/Stingray/ENU/?guid=__stingray_help_managing_content_sjson_html) ?